posthog/models/experiment.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Experiment(models.Model):
+    name: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    filters: models.JSONField = models.JSONField(default=dict)
```

Comment:
Was thinking a bit more about this (ugh, db changes are one place where you got to think about the future)

and: In the future, it's possible that we'll have multiple target metrics (insights) in an Experiment that we want to track.

Hmm, but when (if) the time comes, we can deprecate this. I guess it doesn't make sense to overengineer today.

Reply:
The way I imagine this will work is: `filters` from here are the basic filters, and what we show on the dashboard will have a breakdown applied on top (decided by `feature_flags` ) and have a start date set to whenever users specified during creation. And end_date now().